Clarification of how to use a ScrollViewer (#300)
AvaloniaUI/Documentation: docs/controls/scrollviewer.md

# ScrollViewer

> The `ScrollViewer` control scrolls its content if the content is bigger than the space available. A `ScrollViewer` cannot be contained in a control that has infinate height such as a `StackPanel` but is better contained in a `Panel` with a fixed `Height`.

Reviewer: ```suggestion
The `ScrollViewer` control scrolls its content if the content is bigger than the space available. A `ScrollViewer` cannot be contained in a control that has infinite height or width (depending on scrolling direction) such as a `StackPanel`. 
```

Reviewer: I think it would be more accurate to mention width as well. 
And not sure if we need to mention Panel and fixing the parent's boundaries. In the real app root size is already fixed - Window's client size. And scrollviewer will work there unless there was an ancestor with infinite boundaries.

Author: If you have specific verbiage I'm happy to make the change. All I know is I spent an hour spinning my wheels with this not working before I got lucky that someone clued me in on Gitter.  Just trying to save the next guy some hard ache.

Author: Ah, I see you already made the change!

Author: Hey, I'm not sure how to do this, it appears that the change was approved but that I have the option to "Commit suggestion" which implies that your suggestion is not committed. Also, after letting your suggestion sink in I actually kind of like saying that you can use a "Panel".  The reason why is because as I am sitting here, I couldn't tell you which control are infinite and which are not.  I actually just guessed at "Panel" when I was someone suggested that on Gitter.

Reviewer: You can make any control with limited size by setting height/width or maxheight/maxwidth. Including StackPanel. So that's another way to make ScrollViewer work there.

Reviewer: I have added this detail
